x-pack/plugins/ml/public/application/routing/routes/jobs_list.tsx

```diff
@@ -45,13 +47,12 @@ export const jobListRouteFactory = (navigateToPath: NavigateToPath, basePath: st
 });
 
 const PageWrapper: FC<PageProps> = ({ deps }) => {
-  const { context } = useResolver(
-    undefined,
-    undefined,
-    deps.config,
-    deps.dataViewsContract,
-    basicResolvers(deps)
-  );
+  const { context } = useRouteResolver('full', ['canGetJobs'], {
+    getMlNodeCount,
```

Comment:
Can we use basicResolvers() here?

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75